src/scaladoc/scala/tools/nsc/doc/html/resource/lib/template.js

```diff
@@ -298,13 +287,29 @@ $(document).ready(function() {
         return $(memberSelector);
     }
 
-    // highlight and jump to selected member
+    // highlight and jump to selected member if an anchor is provided
     if (window.location.hash) {
         var jqElem = findElementByHash(window.location.hash);
         if (jqElem.length > 0)
             exposeMember(jqElem);
     }
 
+    $("#template span.permalink").click(function(e) {
```

Comment:
Moved down to here so it will attach to the DOM elements created by `initInherit()`